### PR TITLE
Well i guess there can be a toggle in the respawn to spawn

### PR DESCRIPTION
**Pull request by crazyaccess**

### CODE REVIEW

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ link:
   # Requires link to play on the server
   required: false
 
+  # Make the player respawn to spawn upon getting linked (disabled by default if the link required is set to false)
```
Comment (JSH32) [inline src/main/resources/config.yml]: Make this say

```
# Should the player respawn at the world spawn after verification. This setting will only be used if link.required is true
```

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@ link:
...
+  respawn_to_spawn: false
```
Comment (JSH32) [inline src/main/resources/config.yml]: Change this to verify_spawn

File: src/main/java/com/github/riku32/discordlink/Config.java [hunk 3]
```diff
@@ -46,6 +46,7 @@
     private final String statusDeathUnlinked;
 
     private final boolean linkRequired;
+    private final boolean linkedRespawnToSpawn;
```
Comment (JSH32) [inline src/main/java/com/github/riku32/discordlink/Config.java]: Change to verifySpawn

File: src/main/java/com/github/riku32/discordlink/discord/listeners/VerificationListener.java [hunk 4]
```diff
@@ -108,10 +108,13 @@ public void onButtonClick(ButtonClickEvent e) {
                                                     null, e.getUser().getAvatarUrl())
                                             .build())
                                             .queue();
+                                            return null;
```
Comment (JSH32) [inline src/main/java/com/github/riku32/discordlink/discord/listeners/VerificationListener.java]: I dont think this is needed so remove it

File: src/main/java/com/github/riku32/discordlink/discord/listeners/VerificationListener.java [hunk 5]
```diff
@@ -108,10 +108,13 @@ public void onButtonClick(ButtonClickEvent e) {
...
                             }
 
                             Bukkit.getScheduler().runTask(plugin, () -> {
+                                if (plugin.getPluginConfig().isLinkedRespawnToSpawn()) {
```
Comment (JSH32) [inline src/main/java/com/github/riku32/discordlink/discord/listeners/VerificationListener.java]: Make this isVerifySpawn

File: src/main/java/com/github/riku32/discordlink/Config.java [hunk 6]
```diff
@@ -55,6 +56,7 @@
      */
     public Config(FileConfiguration configuration) throws NoSuchElementException {
         linkRequired = Boolean.parseBoolean(getAsStringNotNull(configuration, "link.required"));
+        if (!linkRequired) { linkedRespawnToSpawn = Boolean.parseBoolean(getAsStringNotNull(configuration, "link.respawn_to_spawn")); };
```
Comment (JSH32) [inline src/main/java/com/github/riku32/discordlink/Config.java]: Make this link.verify_spawn

**Review by JSH32 — COMMENTED** [review]
Fix a few of the problems I lined out and then I'll approve it. 

Note: I would just make them myself but im in another state with a shitty laptop so im slow to these things till the 12th lmao. Doing code review on mobile rn lmao

**Comment by JSH32** [comment]
Also the action java version is wrong so im not gonna run it run. Dont worry abt that

**Comment by JSH32** [comment]
Also more descriptive names for PR and issues in the future please.

**Comment by JSH32** [comment]
Java 11 btw

**Comment by crazyaccess** [comment]
Noice

**Comment by JSH32** [comment]
Yeah fix the naming and the error i put in review and ill merge.

**Comment by JSH32** [comment]
Added two new reviews.

**Comment by crazyaccess** [comment]
2 new reviews ?

**Comment by JSH32** [comment]
Comments. Read the things I just left

**Comment by crazyaccess** [comment]
Oke i will have more descriptive stuff in the future

**Comment by JSH32** [comment]
Nono i mean the code review comments.

**Comment by JSH32** [comment]
Where it says riku started review

**Comment by crazyaccess** [comment]
I will test the artifact produced rn...

**Comment by crazyaccess** [comment]
checked it myself with all the permutations and combinations (well there were just 4 combinations tho)

**Comment by crazyaccess** [comment]
If you by chance merge this one, pls keep it open i will add more enhancement to this one like if the bed / spawn point (those nether spawn pillars) exist then they can optionally have a flag in the config on teleporting there instead of the world spawn or have a completely different teleport local altogether

**Comment by JSH32** [comment]
If you do that make another PR and make it required. Respawn point is ideal

**Pull request merged**